travis_tests.py

```diff
@@ -5,4 +5,6 @@
 import nose, warnings
 with warnings.catch_warnings():
     warnings.simplefilter("ignore")
-nose.main('GPyOpt', defaultTest='GPyOpt/testing/', argv=[''])
\ No newline at end of file
+
+if __name__=='__main__': # for parallel computation
```

Comment:
I don't understand why this change was needed. Can you please elaborate here?

Reply:
This is the reason why multiprocessing is currently not working under windows. On Windows many types from multiprocessing need to be picklable so that child processes can use them. However, one should generally avoid sending shared objects to other processes using pipes or queues. Instead you should arrange the program so that a process which needs access to a shared resource created elsewhere can inherit it from an ancestor process